test/checks/aria/aria-required-attr.js

```diff
@@ -1,48 +1,72 @@
-describe('aria-required-attr', function() {
+describe('aria-required-attr', function () {
```

Comment:
Check didn't change, just want to better prove empty values are not allowed under this check.